### PR TITLE
Change pulling period from 5s to 10s

### PR DESCRIPTION
**Pull request by Freakachoo**
Signed-off-by: Freakachoo <freakachoo.corp@gmail.com>

### CODE REVIEW

File: curiefense/traffic-metrics-exporter/src/traffic_metrics_exporter.py [hunk 1]
```diff
@@ -202,12 +202,11 @@ def get_t2():
...
         try:
             five_sec_t2 = requests.get(config["url"]).content.decode()
+            q.put(five_sec_t2)
         except Exception as e:
             logger.exception(e)
 
-        q.put(five_sec_t2)
-
-        time.sleep(5 - (time.time() - start_time))
+        time.sleep(10 - (time.time() - start_time))
```
Comment (tzuryby) [inline curiefense/traffic-metrics-exporter/src/traffic_metrics_exporter.py:209]: parameterized?

File: curiefense/traffic-metrics-exporter/src/traffic_metrics_exporter.py [hunk 2]
```diff
@@ -202,12 +202,11 @@ def get_t2():
         time.time() - start_time
         try:
             five_sec_t2 = requests.get(config["url"]).content.decode()
+            q.put(five_sec_t2)
```
Comment (tzuryby) [inline curiefense/traffic-metrics-exporter/src/traffic_metrics_exporter.py:205]: funny. the put was after the except. lol.

File: curiefense/traffic-metrics-exporter/src/traffic_metrics_exporter.py (hunk 1 above)
Comment (tzuryby) [inline curiefense/traffic-metrics-exporter/src/traffic_metrics_exporter.py:209]: if processing .put took longer than anticipated, that will be a sleep with negative number.

**Review by tzuryby — CHANGES_REQUESTED** [review]

File: curiefense/traffic-metrics-exporter/src/traffic_metrics_exporter.py (hunk 1 above)
Comment (Freakachoo) [reply curiefense/traffic-metrics-exporter/src/traffic_metrics_exporter.py:209]: True, and on debugging it causes troubles. I'll add protection from this.

Comment (Freakachoo) [reply curiefense/traffic-metrics-exporter/src/traffic_metrics_exporter.py]: done

**Comment by Freakachoo** [comment]
@tzuryby fixed your comments

**Review by tzuryby — APPROVED** [review]

**Pull request merged**